### PR TITLE
Fix pagination issue in ExtensibleTableComponent

### LINKED ISSUE #19194: Pagination doesn't work after filtering in Angular
Related link -> https://github.com/abpframework/abp/pull/18871#pullrequestreview-1912692223

### PR DESCRIPTION
**Pull request by Sinan997**
Resolves #19194 

> Edited by @masumulu28 

@oykuermann this PR is important for all datagrid, can you please make detailed test thank you

### CODE REVIEW

**Review by masum-ulu — APPROVED** [review]

**Review by oykuermann — APPROVED** [review]

File: npm/ng-packs/packages/theme-shared/src/lib/directives/ngx-datatable-list.directive.ts [hunk 1]
```diff
@@ -53,14 +53,6 @@ export class NgxDatatableListDirective implements OnChanges, OnDestroy, OnInit {
...
   }
 
-  private subscribeToPage() {
-    const sub = this.table.page.subscribe(({ offset }) => {
-      this.list.page = offset;
-      this.table.offset = offset;
-    });
-    this.subscription.add(sub);
-  }
-
```
Comment (mustafapsd) [inline npm/ng-packs/packages/theme-shared/src/lib/directives/ngx-datatable-list.directive.ts:63]: This change causing broken pagination now. Whether the `_filter` is empty or not, pagination does not work on the `ngx-datatable` component.

## Reproduce
Just try to paginate any `ngx-datatable` that works with `[list]`, and nothing happens.

@Sinan997 @oykuermann

Comment (Sinan997) [reply npm/ng-packs/packages/theme-shared/src/lib/directives/ngx-datatable-list.directive.ts:63]: We have moved pagination functionality to ngx-datatable's `input, output`.
https://github.com/abpframework/abp/pull/19203/files#diff-0c5913f6f68aa1645a0b5aa522f62c0b7e5f48657e8398c2f625926b138f847dR6-R7

So if you are using ngx-datatable directly. You should add the code below to your ngx-datatable element

```ts
[offset]="list?.page"
(page)="list.page = $event.offset"
```

Comment (mustafapsd) [reply npm/ng-packs/packages/theme-shared/src/lib/directives/ngx-datatable-list.directive.ts:63]: I think a patch version (8.0.x) should not contain a breaking change. I'm sure you can find a way that supports both.

Comment (Sinan997) [reply npm/ng-packs/packages/theme-shared/src/lib/directives/ngx-datatable-list.directive.ts:63]: Yes, you are right. It's a breaking change. I didn't realise that people uses ngx-datatable directly.

Thanks for your attention.

**Pull request merged**